src/components/patterns/PricingTable/PricingTable.js

```diff
@@ -48,15 +50,22 @@ const PricingLink = ({ children, ...props }) => {
 
 const PricingBox = props => (
   <Box
-    borderRadius={3}
-    textAlign='center'
-    width={['100%', '100%', 8, 8]}
-    px={[3, 3, 5, 5]}
-    py={[4, 4, 5, 5]}
-    my={[4, 4, 0, 0]}
-    border={1}
-    bg='white'
     {...props}
+    css={`
+      ${theme({
+        borderRadius: 3,
+        textAlign: 'center',
+        width: ['100%', '100%', 8, 8],
+        px: [3, 3, 5, 5],
+        py: [4, 4, 5, 5],
+        my: [4, 4, 0, 0],
+        border: 1,
+        bg: 'white'
+      })}
+
+      /* TODO: How do we pass styles in? This doesn't seem to work... */
```

Comment:
@Kikobeats the pricing table is pretty broken, from what I can tell it's due to the mix of `styled()` and the `css` prop. Any idea how to solve this?

Reply:
yea you can't mix both together, I believe you have to do something like this:

```jsx
const PricingBox = props => (
  <Box
    {...props}
    css={`
      ${theme({
        borderRadius: 3,
        textAlign: 'center',
        width: ['100%', '100%', 8, 8],
        px: [3, 3, 5, 5],
        py: [4, 4, 5, 5],
        my: [4, 4, 0, 0],
        border: 1,
        bg: 'white',
        ...props.css
      })}
    `}
  />
)

const FreePricingBox = styled(PricingBox)({
  css: {
    borderColor: 'colors.black20'
  }
})
```